wnfs/src/private/forest.rs

```diff
@@ -140,24 +140,19 @@ impl PrivateForest {
         Ok(pair.map(|p| p.value))
     }
 
-    /// Returns a stream of all private nodes that could be decrypted at given revision.
-    ///
-    /// The stream of results is ordered by CID.
-    ///
-    /// Each item in the resulting stream represents an instance of a concurrent write.
-    pub fn get_multivalue<'a>(
+    pub(crate) fn get_multivalue_with_revision_ref<'a>(
         &'a self,
-        revision: &'a RevisionRef,
+        revision_ref: RevisionRef,
```

Comment:
Yeah if we make this private, that may be an opportunity to rethink the revision ref again.

I think we can remove it, we practically pretty much only need it for this place in the API.
And internally, it's not too bad if instead of `RevisionRef` we need to pass `TemporalKey` and `HashOutput` as separate arguments.

I guess in the future getting multivalues should not be part of the API, instead (when we have conflict reconciliation) you'll only work with `AccessKey` and it'll merge it with the other multivalues in the same revision automatically.

Anyway, I don't know what the right move atm is. Perhaps we can pub(crate) the mutlivalue API completely?

In any case, I'm not even that sure, so that could happen in a separate PR.

Reply:
Even if we do have conflict reconciliation, I think `get_multivalue` is a useful thing when you need to know what the conflicts are.

I agree RevisionRef feels unnecessary at this point. Let me see if I can get rid of it entirely.

Reply:
Decided to leave RevisionRef to a separate PR because it requires possibly deeper changes to how privateref is derived.